docs/reset-security-verification-for-wog-account.md

```diff
@@ -2,65 +2,86 @@
 
-**Audience**
+## Audience
 
-- Users who have a WOG profile(account)
-- If you have a TechPass account and its domain is not ```techpass.gov.sg```.
+Users who have a WOG profile(account) may want to reset the MFA of their WOG account for any of the following reasons:
 
->**Note**:
->
->- You must [Create service request](https://go.gov.sg/techpass-sr) to remove the MFA of your TechPass profile.
->
->- When the request is processed successfully, you will be able to register a new TechPass MFA profile when you try to sign in again.
->
->- If you have changed your mobile phone, use your new device to complete these steps. 
+- Changed mobile phone.
+- Lost mobile phone.
+- Deleted the Authenticator app from the mobile phone or unable to access the app on your phone.
+- Change in the organisational email address linked to the WOG account.
 
-**Preqrequisites** 
+?> You may also need to [reset MFA of your TechPass account](reset-security-verification-for-wog-account).
 
-If you have changed the mobile phone and want to reset your WOG MFA on the new mobile phone:
+## Prerequisites
 
-- You must have the old mobile phone.
-- You must be able to authenticate your WOG login using the old mobile phone.
+- You must have the old mobile phone to authenticate your WOG login.
 
-If you have lost your old mobile phone, formatted the old phone(factory reset),or unable to authenticate your log in using the old mobile phone, contact your Agency Facility Management(AFM) to remove the MFA configured for your WOG account.
+?> If you have lost your old mobile phone, formatted the old phone(factory reset),or unable to authenticate your log in using the old mobile phone, contact your Agency Facility Management(AFM) to remove the MFA configured for your WOG account. They will get provide the steps to reset your WOG MFA and also your TechPass MFA.
 
+### To reset MFA for WOG account
 
- <!--Public officers might have to reset security verification for their WOG account in the following cases:
+1. Go to [Microsoft Security info](https://mysignins.microsoft.com/security-info).
+2. If prompted, sign in to your WOG account.
+3. On the **Security info** page, click **+ Add sign-in method**.
+![add-sign-in-method](assets/images/reset-techpass-mfa-vendor/add-sign-in-method.png)
+4. In **Add a method**, select **Authenticator app** and click **Add**.
+![add-auth-method](assets/images/reset-techpass-mfa-vendor/add-method.png)
+5. Click **Next**.
+![install-auth-method](assets/images/reset-techpass-mfa-vendor/install-auth-app.png)
+6. Follow the on-screen instructions to add your TechPass account and click **Next**.
+![keep-your-account-secure-next](assets/images/onboarding/po-non-se/keep-your-account-secure-next.png)
+7. Download and install Microsoft Authenticator on your new[Microsoft phone](https://www.microsoft.com/en-sg/store/apps/windows-phone), [Android](https://play.google.com/store/apps?hl=en&amp;gl=US) or [iOS phone](https://www.apple.com/app-store/).
```

Comment:
Add a space between **new** and **Microsoft**.

Reply:
ty for noticing it.